packages/flutter_test/lib/src/binding.dart

```diff
@@ -106,6 +111,14 @@ mixin TestDefaultBinaryMessengerBinding on BindingBase, ServicesBinding {
   }
 }
 
+/// An accessibility announcement.
+class Announcement {
+  Announcement(this.message, this.textDirection, {this.assertiveness = Assertiveness.polite});
+  String message;
+  TextDirection textDirection;
+  Assertiveness? assertiveness;
```

Comment:
The fields of this class should probably be `final` and the constructor `const`.